.circleci/config.yml

```diff
@@ -152,7 +152,8 @@ jobs:
             - run:
                 name: Check for duplicated packages
                 command: |
-                  if [[ $(git diff --name-status master | grep pnpm.lock) == "" ]];
+                  # #default-branch-switch
+                  if [[ $(git diff --name-status next | grep -E 'pnpm-workspace\.yaml|pnpm-lock.yaml|package\.json') == "" ]];
```

Comment:
Should this be `next`? Isn't the default branch `master`?

Reply:
copy-pasta... 😅